src/vnsw/agent/contrail/windows/msi/agent_msi.wxs

```diff
@@ -27,6 +27,9 @@
                             <ServiceInstall Id="CreateService" Name="ContrailAgent" DisplayName="ContrailAgent" Type="ownProcess" Start="demand" ErrorControl="normal"/>
                             <ServiceControl Id="DeleteService" Name="ContrailAgent" Remove="both" Stop="both" Wait="yes"/>
                         </Component>
+                         <Component Id="ContrailVrouterApi" Guid="21F50C2B-7F38-493B-B545-EC7DED2CB2F7" Win64="yes">
+                            <File Id="ContrailVrouterApi" Source="build\noarch\contrail-vrouter-api\dist\contrail-vrouter-api-1.0.tar.gz"/>
```

Comment:
should it be copied here for sure?

Reply:
I will check if it is easy to do.

Reply:
And use '/' instead of '\\'

Reply:
Unfortunately WiX doesn't have straightforward solution to use this file from temporary directory.